orchestra/utils/task_lifecycle.py

```diff
@@ -541,6 +541,9 @@ def tasks_assigned_to_worker(worker):
 
     tasks_assigned = []
     time_now = timezone.now()
+    pending_todos_filter = Q(status=Todo.Status.PENDING.value)
+    non_template_todo_filter = Q(template=None)
```

Comment:
minor, feel free to not address it since it's not a bug: consider `template__isnull=True` for consistency. they are equivalent, so no worries if you want to skip this for now.

Reply:
Fixed!